src/main/java/bio/terra/testrunner/runner/TestRunner.java

```diff
@@ -394,6 +338,39 @@ private void executeTestConfigurationNoGuaranteedCleanup() throws Exception {
     }
   }
 
+  private VersionScript instantiateVersionScript(
+      VersionScriptSpecification versionScriptSpecification) {
+    try {
+      VersionScript versionScript =
+          versionScriptSpecification.scriptClass.getDeclaredConstructor().newInstance();
+      // set any parameters specified by the configuration
+      versionScript.setParameters(versionScriptSpecification.parameters);
+
+      // call the determineVersion method to get the version
+      logger.info("Version: Calling {}.determineVersion()", versionScript.getClass().getName());
+      return versionScript;
+    } catch (InstantiationException | IllegalAccessException e) {
+      logger.error(
+          "Version: Error calling constructor of VersionScript class: {}",
+          versionScriptSpecification.name,
+          e);
+      throw new IllegalArgumentException(
+          "Error calling constructor of VersionScript class: " + versionScriptSpecification.name,
+          e);
+    } catch (Exception e) {
+      throw new RuntimeException(e);
```

Comment:
I don't think you need to wrap in a `RuntimeException` here. Can you just call the `setParameters` inline above?

Reply:
Simplified code with a for-loop.